src/components/Filters/CollapsibleLabel.tsx

```diff
@@ -1,6 +1,17 @@
 import classNames from 'classnames';
 import { ChevronIcon } from '../../icons/ChevronIcon';
 import { useFilterGroupContext } from './FilterGroupContext';
+import { useComposedCssClasses } from '../../hooks/useComposedCssClasses';
+
+/**
+ * The CSS class interface for {@link CollapsibleLabelProps}.
+ *
+ * @public
+ */
+export interface CollapsibleLabelCssClasses {
+  button?: string,
+  titleLabel?: string
```

Comment:
it feels a little weird for this to be named `titleLabel` in terms of the CSS classes for a collapsible label. for this component, it's simply it's label. it doesn't have the notion of it being a "title" label. can this just be called `label`?